servers/republik/migrations/sqls/20181119124113-cancellations-up.sql

```diff
@@ -0,0 +1,30 @@
+CREATE DOMAIN cancel_category AS TEXT
+  CHECK(
+    VALUE IN ('EDITORIAL', 'NO_TIME', 'VAUNT', 'LOGIN', 'PAPER', 'NO_AUTO_RENEWAL', 'TOO_EXPENSIVE', 'OTHER')
```

Comment:
`VAUNT` is missing in schema declaration. And quite frankly, not sure what it even is.

Reply:
good catch. `VAUNT` was `Selbstbeweihräucherung`, but it got kicked during the development.